index.bs

```diff
@@ -7155,6 +7155,45 @@ partial dictionary MLOpSupportLimits {
     ::
         The interpolation algorithm used to fill the output tensor values.
 
+        Both algorithms start with these inputs, computed for each spatial axis:
+        ```
+        scale = outputSize / inputSize
+        unclampedCoordinate = (outputCoordinate + 0.5) / scale - 0.5
+        inputCoordinate = max(min(unclampedCoordinate, inputSize - 1), 0)
```

Comment:
Looks correct 👍, and this is even generally extensible to y and z too, if it was resampleND (to simultaneously resampling the channel or batch dimension too).

```suggestion
        inputCoordinate = clamp(unclampedCoordinate, 0, inputSize - 1)
```